### PR TITLE
relaxed tests

### PR DESCRIPTION
**Pull request by Ilya-Krylov**

### CODE REVIEW

**Comment by Ilya-Krylov** [comment]
run ote_sdk tests

**Comment by nervana-ff** [comment]
Can one of the admins verify this patch?

**Comment by Ilya-Krylov** [comment]
run ote_sdk tests

File: tests/ote_cli/test_ote_cli_tools_anomaly_classification.py [hunk 1]
```diff
@@ -132,7 +132,7 @@ def test_nncf_eval(self, template):
         if template.entrypoints.nncf is None:
             pytest.skip("nncf entrypoint is none")
 
-        nncf_eval_testing(template, root, ote_dir, args)
+        nncf_eval_testing(template, root, ote_dir, args, threshold=0.1)
```
Comment (alexsu52) [inline tests/ote_cli/test_ote_cli_tools_anomaly_classification.py:135]: It looks like we can use `ote_eval_testing` w/o threshold. @Ilya-Krylov, @evgeniya-egupova what do you think?

Comment (Ilya-Krylov) [reply tests/ote_cli/test_ote_cli_tools_anomaly_classification.py:135]: ote_eval_testing or nncf_eval_testing ?

w/o threshold is threshold=0 or threshold=infinity ??

Comment (alexsu52) [reply tests/ote_cli/test_ote_cli_tools_anomaly_classification.py:135]: ote_eval_testing

**Pull request merged**